controllers/src/main/scala/cromwell/pipeline/controller/ProjectFileController.scala

```diff
@@ -35,6 +37,17 @@ class ProjectFileController(wdlService: ProjectFileService)(implicit val executi
             }
           }
         }
+      },
+      path("files" / "inputs") {
+        post {
+          entity(as[FileContent]) { request =>
+            onComplete(wdlService.getInputs(request)) {
```

Comment:
Why we use method getInputs in POST method?

Reply:
Because this request contains body, and get with body isn't a good idea